modular_citadel/code/game/gamemodes/gangs/dominator.dm

```diff
@@ -0,0 +1,241 @@
+#define DOM_BLOCKED_SPAM_CAP 6
+#define DOM_REQUIRED_TURFS 30
+#define DOM_HULK_HITS_REQUIRED 10
+
+/obj/machinery/dominator
+	name = "dominator"
+	desc = "A visibly sinister device. Looks like you can break it if you hit it enough."
+	icon = 'icons/obj/machines/dominator.dmi'
+	icon_state = "dominator"
+	density = TRUE
+	anchored = TRUE
+	layer = HIGH_OBJ_LAYER
+	max_integrity = 300
+	integrity_failure = 100
+	armor = list("melee" = 20, "bullet" = 50, "laser" = 50, "energy" = 50, "bomb" = 10, "bio" = 100, "rad" = 100, "fire" = 10, "acid" = 70)
+	var/datum/team/gang/gang
+	var/operating = FALSE	//false=standby or broken, true=takeover
+	var/warned = FALSE	//if this device has set off the warning at <3 minutes yet
+	var/spam_prevention = DOM_BLOCKED_SPAM_CAP //first message is immediate
+	var/datum/effect_system/spark_spread/spark_system
+	var/obj/effect/countdown/dominator/countdown
+
+/obj/machinery/dominator/Initialize()
+	. = ..()
+	set_light(2)
```

Comment:
Light power reduced to 0.75, Light range 2 kept, the mind dominator has a white light tone along with the monitor since it has no team tied to it yet though.